clients/python/tracker_client/queries.py

```diff
@@ -29,6 +29,10 @@
                 ... on AuthResult {
                     authToken
                 }
+                ... on SignInError{
```

Comment:
This may be better off in a separate PR, however we may want to add support in for users who have tfa enabled.
```graphql
result {
  ... on TFASignInResult {
    sendMethod
    authenticateToken
  }
  ... on AuthResult {
    authToken
  }
  ... on SignInError {
    code
    description
  }
}
```

Reply:
I'll add it to the project board, I think that should definitely be added but in another PR.